src/System.Memory/src/System/ThrowHelper.cs

```diff
@@ -60,10 +60,18 @@ internal static class ThrowHelper
         [MethodImpl(MethodImplOptions.NoInlining)]
         private static Exception CreateArgumentOutOfRangeException_SymbolDoesNotFit() { return new ArgumentOutOfRangeException("symbol", SR.Argument_BadFormatSpecifier); }
 
+        internal static void ThrowInvalidOperationException() { throw CreateInvalidOperationException(); }
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        private static Exception CreateInvalidOperationException() { return new InvalidOperationException(); }
+
         internal static void ThrowInvalidOperationException_OutstandingReferences() { throw CreateInvalidOperationException_OutstandingReferences(); }
         [MethodImpl(MethodImplOptions.NoInlining)]
         private static Exception CreateInvalidOperationException_OutstandingReferences() { return new InvalidOperationException(SR.OutstandingReferences); }
 
+        internal static void ThrowObjectDisposedException(string objectName) { throw CreateObjectDisposedException(objectName); }
```

Comment:
We recently removed the objectName argument from ThrowObjectDisposedException_MemoryDisposed.
https://github.com/dotnet/corefx/commit/3f0381475cb9f228cdc7ab5bdc12b13220d96776#diff-4021c6d7a5c4f084b4e8867d8cb0f973L67
@jkotas, why was that?

Reply:
https://github.com/dotnet/coreclr/pull/15909#discussion_r162393699

Reply:
Thanks for the context! Should we keep passing the object name to the newly added ThrowObjectDisposedException method or remove that argument here as well?